docs/04-operation-guides/troubleshooting/01-k3d-fails-on-linux.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Provisioning k3d fails on a Linux machine
+---
+
+## Symptom
+
+You're on a Linux machine and provisioning k3d fails with a message like `Cannot bind to reserved port 80`.
+
+## Cause
+
+By default, provisioning tries to use ports 80 and 433.
+On Linux, the port is reserved.
```

Comment:
"reserved" is a bit inaccurate. How about "..., the ports are reserved to be used by a privileged user."?